crates/y-sweet/src/server.rs

```diff
@@ -145,14 +271,14 @@ impl Server {
         Ok(self
             .docs
             .get(doc_id)
-            .expect("Doc should exist, we just created it.")
+            .ok_or_else(|| anyhow!("Failed to get-or-create doc"))?
```

Comment:
Maybe we can use `.context()` from `anyhow` here, so that any error details are propagated?